shared/keystore/keystore.go

```diff
@@ -104,7 +105,10 @@ func (ks Store) GetKeys(directory, fileprefix, password string) (map[string]*Key
 			}
 			key, err := DecryptKey(keyjson, password)
 			if err != nil {
-				return nil, err
+				if warnOnFail {
```

Comment:
Why is this a toggleable action?

Reply:
`GetKeys` is called in various places, including to check if certain keys exist, so this flag stops spurious errors from showing up in the logs.